src/nora/components/NoraButton/NoraButton.module.scss

```diff
@@ -0,0 +1,17 @@
+.Button {
+  font-weight: 500;
+  font-size: 13px;
+  height: var(--Space-32);
+  line-height: var(--Space-32);
+  letter-spacing: 0.015em;
+  color: white;
+  background: var(--NoraBlue-400);
+  border-radius: 3px;
+  text-align: center;
+  display: inline-block;
+}
+
+/* Used only for md example */
```

Comment:
You can create an `example.scss` file and pull it in via es6 import right from the .md example. We've opted for that to keep example css out of the library ones

Reply:
Oh cool! Didn't see that